### PR TITLE
LPS-159240

### PR DESCRIPTION
**Pull request by felipelorenz**

### CODE REVIEW

**Comment by felipelorenz** [comment]
ci:test:stable

**Comment by felipelorenz** [comment]
ci:test:relevant

File: modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java [hunk 1]
```diff
@@ -544,6 +571,19 @@ private void _setUpThemeDisplay() {
 		);
 	}
 
+	private void _stubSearchBarPortletDisplayContextFactory(
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: I would rename that method to `_setUpSearchBarPortletDisplayContextFactory()` or `_mockSearchBarPortletDisplayContextFactory()`

@felipelorenz @AlmirFe

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: And would create the inside `SearchBarPortletDisplayContextFactory` the method

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: Better than that is use the method that already exists to add these changes, you could add that in `_createSearchBarPortletDisplayContextFactory(x, x, x);`

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: That change will be cleaner than add a new methods

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: That change will be cleaner than add a new methods

Comment (gustavolimav) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: @felipelorenz @AlmirFe let me know if you have some doubt about that

Comment (felipelorenz) [reply modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/search/bar/portlet/display/context/factory/SearchBarPortletDisplayContextFactoryTest.java:574]: Open a new Pull Request #518

**Pull request closed without merging**